src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -1112,7 +1113,6 @@ class LibraryServiceGapicClient
      * ```
      * $libraryServiceClient = new LibraryServiceClient();
      * try {
```

Comment:
I wonder whether we should have a blank line before these `try`s. WDYT?

Not a big deal right now, we can tweak this later.